schema/training-course.js

```diff
@@ -21,7 +22,7 @@ class TrainingCourse extends BaseModel {
         },
         startDate: { type: ['string', 'null'], pattern: date.yearMonthDay },
         title: { type: 'string' },
-        coursePurpose: { type: 'string', enum: ['higher-education', 'training'] },
+        coursePurpose: { type: 'string', enum: [trainingCoursePurpose.higherEducation, trainingCoursePurpose.training] },
```

Comment:
```suggestion
        coursePurpose: { type: 'string', enum: Object.keys(trainingCoursePurpose) },
```

Undo change to asl-metrics